packages/vapor/scss/components/home-card.scss

```diff
@@ -16,7 +16,7 @@
     margin-bottom: $home-card-margin;
     padding: 2 * $spacing;
     background: var(--pure-white);
-    border: 1px solid var(--home-card-color);
+    border: 1px solid rgba($grey-5, $transparency-4);
```

Comment:
grey-5 is an old color right? :eyes:

Reply:
Yes, a lot of colors from the old palette are still used, they were not in the scope of the light reskin project.